locations/spiders/serious_coffee.py

```diff
@@ -0,0 +1,128 @@
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+DAY_MAPPING = {
+    'Monday': 'Mo',
+    'Tuesday': 'Tu',
+    'Wednesday': 'We',
+    'Thursday': 'Th',
+    'Friday': 'Fr',
+    'Saturday': 'Sa',
+    'Sunday': 'Su'
+}
+
+
+class SeriousCoffeeSpider(scrapy.Spider):
+
+    name = "seriouscoffee"
+    allowed_domains = ["www.seriouscoffee.com"]
+    download_delay = 0.5
+    start_urls = (
+        'http://www.seriouscoffee.com/locations/',
+    )
+
+    def parse_day(self, day):
+        if re.search('-', day):
+            days = day.split('-')
+            osm_days = []
+            if len(days) == 2:
+                for day in days:
+                    osm_day = DAY_MAPPING[day.strip()]
```

Comment:
Instead of using a mapping dictionary like this you can just call `day.strip().title()[:2]`.